### PR TITLE
wank

### PR DESCRIPTION
**Pull request by sargeant45**
wank forever

- wank

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1 +1,10 @@
-wank
+<script>
+wAnk = 100;
+window.setInterval(function() {
+  wank = "wank " + document.body.innerHTML;
```
Comment (sargeant45) [inline index.html]: wank

**Comment by sargeant45** [comment]
wank

**Comment by probablyjosh** [comment]
@sargeant45 wank

**Comment by sargeant45** [comment]
👉💦 wank
On Fri, Jul 24, 2015 at 3:34 AM Josh Trommel <notifications@github.com>
wrote:

> @sargeant45 <https://github.com/sargeant45> wank
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/probablyjosh/wank/pull/2#issuecomment-124468792>.
>
-- 
- Ethan

**Pull request merged**